### PR TITLE
feat: adding new testing library exkorpion

### PR DESCRIPTION
**Pull request by ivancorrales**
Make sure you have the following format

##Title

Add Package "name-of-the-package"

##Description

Resolves "#task-number"

##Commit message
Add name-of-the-package

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1077,7 +1077,6 @@ A curated list of amazingly awesome Elixir libraries, resources, and shiny thing
 
 ## Testing
 *Libraries for testing codebases and generating test data.*
-
```
Comment (fladson) [inline README.md]: We follow a empty line pattern after the categories. Can you edit this? 😁

Comment (ivancorrales) [inline README.md]: Sure, I just did it,

Comment (fladson) [inline README.md]: Thanks and another thing, sorry I didn't notice before, please follow the repo description, that is `A BDD library for Elixir developers`

Comment (ivancorrales) [inline README.md]: Done!,

**Pull request merged**